libs/domains/services/feature/src/lib/hooks/use-count-services/use-count-services.ts

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from '@tanstack/react-query'
+import { queries } from '@qovery/state/util-queries'
+
+export interface UseCountServicesProps {
+  environmentId?: string
+}
+
+export function useCountServices({ environmentId }: UseCountServicesProps) {
+  const queryData = useQuery({
+    ...queries.services.list(environmentId!),
+    enabled: Boolean(environmentId),
```

Comment:
use react-query `select` and return the `queryData.data?.length` in it so no need to introduce new `queryData` variable